aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -44,6 +67,24 @@ module aptos_framework::staking_config {
         voting_power_increase_limit: u64,
     }
 
+    /// Staking reward configurations that will be stored with the @aptos_framework account.
+    struct StakingRewardsConfig has copy, drop, key {
+        // The target rewards rate given out every epoch. This will be divided by the rewards rate denominator.
+        // For example, 0.001% (0.00001) can be represented as 10 / 1000000.
+        rewards_rate: u64,
+        // The minimum threshold for rewards_rate. rewards_rate won't be lower than this.
+        // This will be divided by the rewards rate denominator.
+        min_rewards_rate: u64,
+        rewards_rate_denominator: u64,
+        // Reward rate decreases every rewards_rate_period_in_micros microseconds.
+        // Currently it can only equal to 1 year. Keep this field in case we need to change the interval.
+        rewards_rate_period_in_micros: u64,
+        // Timestamp of start of last rewards period.
+        last_rewards_rate_period_start_in_micros: u64,
+        // Rate of reward rate decrease in BPS. 1 bps(basis points) = 0.01%.
+        rewards_rate_decrease_rate_bps: u64,
```

Comment:
Might as well use fixedpoint64 here? You can then use math_fixed: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-stdlib/sources/math_fixed.move (Need to add support for fixedpoint64)